modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -954,32 +954,48 @@ public Object getSystemObjectEntry(
 	@Override
 	public Page<ObjectEntry> getVersionedObjectEntries(
 			DTOConverterContext dtoConverterContext, long objectEntryId,
-			Pagination pagination)
+			String search, Pagination pagination, Sort[] sorts)
 		throws Exception {
 
 		com.liferay.object.model.ObjectEntry serviceBuilderObjectEntry =
 			objectEntryLocalService.getObjectEntry(objectEntryId);
 
-		return Page.of(
-			TransformUtil.transform(
-				_objectEntryVersionService.getObjectEntryVersions(
-					objectEntryId, _getStartPosition(pagination),
-					_getEndPosition(pagination)),
-				objectEntryVersion -> _objectEntryDTOConverter.toDTO(
-					_getObjectEntryVersionDTOConverterContext(
-						dtoConverterContext, objectEntryVersion,
-						serviceBuilderObjectEntry),
-					serviceBuilderObjectEntry)),
-			pagination,
-			_objectEntryVersionService.getObjectEntryVersionsCount(
-				objectEntryId));
+		return SearchUtil.search(
```

Comment:
Same thing related to this [comment](https://github.com/liferay-bpm/liferay-portal/pull/4841/commits/512210f3a9dc7c454176b3748f044fe444f50edb#r2250308183).